src/js/components/CompleteYourProfile/HowItWorksWizard.jsx

```diff
@@ -1,21 +1,56 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
+import TagManager from 'react-gtm-module';
 import Colors from '../../common/components/Style/Colors';
 import normalizedImagePath from '../../common/utils/normalizedImagePath';
 import HowItWorksStep from './Step';
 import DesignTokenColors from '../../common/components/Style/DesignTokenColors';
+import VoterStore from '../../stores/VoterStore';
+import lookupPageNameAndPageTypeDict from '../../utils/lookupPageNameAndPageTypeDict';
 
 const crossIcon = normalizedImagePath('../../../img/global/svg-icons/cross.svg');
 
 
 const HowItWorksWizard = ({ steps, activeStep }) => {
   const [showHowItWorksWizard, setShowHowItWorksWizard] = useState(true);
 
-  const hideHowItWorksWizard = () => {
+  const hideHowItWorksWizard = (buttonId) => {
     setShowHowItWorksWizard(false);
+    // console.log('HowItWorksWizard props:', { steps, activeStep });
+    // dataLayer tracking
+    const { location: { pathname: currentPathname } } = window;
+    // console.log('Current pathname:', currentPathname);
+    const currentPage = lookupPageNameAndPageTypeDict(currentPathname);
+
+    const dataLayerObject = {
+      actionDetails: {
+        actionType: 'close',
+        buttonId,
+      },
+      event: 'action',
+      pageDetails: {
+        PageName: 'HowItWorksWizard',
+        PageType: currentPage.pageType,
```

Comment:
You can fix in follow-up pull request, but these should start with a lowercase letter:
pageName
pageType